.asf.yaml

```diff
@@ -39,8 +39,15 @@ github:
       required_pull_request_reviews:
         required_approving_review_count: 1
 
-# publishes the content of the `asf-site` branch to
-# https://datafusion.apache.org/blog
-publish:
-  whoami: asf-site
-  subdir: blog
+# TODO: After merging into main verify it is safe to remove
+# This section for publish
+# # publishes the content of the `asf-site` branch to
+# # https://datafusion.apache.org/blog
+# publish:
+#   whoami: asf-site
+#   subdir: blog
+
+staging:
+  whoami: asf-break-test
```

Comment:
what does "asf-break-test" mean? Maybe we can add a link here to the ASF instructions

Reply:
I think this was a mistake on my part. The instructions weren't super clear, but I think this has to be `asf-site`. I've updated it in a recent push.